tms/across.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: tms
+title: Across
+description: The Across machine translation integrations
+id: across
+parent: TMSs
+type:
+- tms
+- cat
+tms_url: https://www.across.net/
+api_integrations:
+- slug: deepl
+  formality: true
+  name: DeepL
+- slug: google
+  name: Google Translate
+- slug: kantanmt
+  name: KantanMT
+- slug: language-wire
+  name: Language Wire
+- slug: octavemt
+  name: OctaveMT
+- slug: promt
+  name: PROMT
+- slug: systran
+  name: SYSTRAN
+- slug: textshuttle
+  name: TextShuttle
+fuzzy_repair: true
```

Comment:
I have mixed feelings about adding fuzzy_repair and especially quality_estimation here.

Rather than kinda treating them as a separate product that this TMS happens to integrate, the way that we do for MT.

e.g. KantanQES is not too different from Crowdin MT or LanguageWeaver, in how it relates to TMSes.